### ISSUE TITLE
[EN] Feedback for: /fundamentals/getting-started/web-starter-kit/development-phases.html

### ISSUE THREAD

**Issue by irwind65** [google/WebFundamentals#1206]
I dutifully downloaded all the tools suggested by the Google tutorial, including Sublime 3. My first surprise was that after installing, configuring, and adding plugins to Sublime, and spending about an hour using the editor, I got a popup saying I needed a license. I was taken to a website where a $70 fee was required. That would have been nice to know at the start.

Second, there's a lot of tedious work required to setup Gulp, and it seems not quite ready for prime time. Further, the instructions for setting up and using gulp are either incomplete or inaccurate. I've tried several times to install gulp globally on my Windows 7 system, but I'm unable to execute something as simple as 'gulp -v'. The instructions don't say exactly where this command should be entered. I did find that if I cd to the .bin directory beneath node_modules, the command is recognized by the system, but that is different from functioning globally (or maybe I misunderstand the idea of global). 

Bottom line, more work is needed on the tutorial/instructions to make it bullet proof.

**Comment by hebbet** [comment]
you can use sublime for free, it will only ask every 20th save for a licence.
the website of sublime text also states that it costs 70$

**Comment by petele** [comment]
Thanks for the feedback!
